ui/app/mixins/policy-edit-controller.js

```diff
@@ -27,18 +27,13 @@ export default Mixin.create({
       let flash = this.flashMessages;
       let policyType = model.get('policyType');
       let name = model.get('name');
-      model
-        .save()
-        .then((m) => {
-          flash.success(`${policyType.toUpperCase()} policy "${name}" was successfully saved.`);
-          if (this.wizard.featureState === 'create') {
-            this.wizard.transitionFeatureMachine('create', 'CONTINUE', policyType);
-          }
-          return this.transitionToRoute('vault.cluster.policy.show', m.get('policyType'), m.get('name'));
-        })
-        .catch((e) => {
-          model.set('errors', e.errors);
```

Comment:
The `errors` prop on Model is a getter now and attempting to set it throws an error. In this case the adapter sets the errors automatically on failure so this is not necessary.